### PR TITLE
QuestionnaireResponse remove beta tag

### PR DESCRIPTION
**Pull request by sydneanschutz**
Description
----
QuestionnaireResponse remove beta tag

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Screen Shot 2021-08-19 at 8 17 18 AM
Screen Shot 2021-08-19 at 8 29 04 AM
Screen Shot 2021-08-19 at 8 17 31 AM

### CODE REVIEW

File: content/millennium/r4/clinical/diagnostics/questionnaire-response.md [hunk 1]
```diff
@@ -4,14 +4,13 @@ title: QuestionnaireResponse | R4 API
...
 
-<%= beta_tag %>
-
 * TOC
 {:toc}
 
 ## Overview
 
 The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. A given QuestionnaireResponse must belong to a corresponding Questionnaire. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history is supported.
+The resource should be leveraged as a snapshot in time and new data should be consistently retrieved through the API rather than stored within an application.
```
Comment (sydneanschutz) [inline content/millennium/r4/clinical/diagnostics/questionnaire-response.md:13]: I dont even know if this is true, i just know the item.id might change! and the id-modifiable extension might change too

File: content/millennium/r4/clinical/diagnostics/questionnaire-response.md [hunk 2]
```diff
@@ -130,12 +128,15 @@ The common [errors] and [OperationOutcomes] may be returned.
...
-<%= beta_tag(action: true) %>
-
 Update an QuestionnaireResponse.
 
     PUT /QuestionnaireResponse/:id
 
+_Implementation Notes_
+
+* When an item is returned with the is-modifiable extension as false, that item cannot be updated.
+* An update should only be preformed directly after a QuestionnaireResponse is retrieved. If an item.id is provided from the get operation, it MUST be provided on the subsequent update and must match the item.id returned from the get.
```
[the remark this answers is not in the document]
Comment (MadhurRaj) [reply content/millennium/r4/clinical/diagnostics/questionnaire-response.md:138]: I think it's fine. The table can contain a summarized note that outlines the high-level use of the field.

**Review by MadhurRaj — APPROVED** [review]

**Review by sonikrish — APPROVED** [review]

**Review by bh021790 — APPROVED** [review]

File: content/millennium/r4/clinical/diagnostics/questionnaire-response.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sydneanschutz) [reply content/millennium/r4/clinical/diagnostics/questionnaire-response.md:138]: updated wording with Brittany: https://github.com/cerner/fhir.cerner.com/pull/723/commits/896ec1e5484335b5dff756b68adb5880e7f7cbbb#diff-65d395ab3c58d2ca0d3ac40728ec9e34347612b86c616594927d663c0ccb1c31L137

**Review by RayGunY — APPROVED** [review]

**Comment by sonikrish** [comment]
Screenshot 2021-08-23 at 6 59 06 PM
Screenshot 2021-08-23 at 6 59 15 PM

**Pull request merged**